cmd/host/config_windows.go

```diff
@@ -71,20 +71,23 @@ func validateSubnet(s string) (*subnet, error) {
 
 func gatewayIP(ip net.IP) string {
 	// Gateway is always x.x.x.1
-	return net.IPv4(ip[0], ip[1], ip[2], 1).String()
+	gatewayLastBit := 1
+	return net.IPv4(ip[0], ip[1], ip[2], byte(gatewayLastBit)).String()
 }
 
 func staticDHCP(ip net.IP) map[string]string {
 	// Static DHCP Lease is always x.x.x.2
-	tapDevIP := net.IPv4(ip[0], ip[1], ip[2], 2).String()
+	staticDHCPLastBit := 2
```

Comment:
A bit doesn't go up to 2. Presumably you mean `staticDHCPLastByte`, or maybe `staticDHCPSuffix`.

Reply:
Thanks, I meant `byte` of course :b